app/views/statistics/statistics.mobile.haml

```diff
@@ -0,0 +1,24 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
+-#   licensed under the Affero General Public License version 3 or later.  See
+-#   the COPYRIGHT file.
+%header
+
+.row-fluid
+  %h1
+    = t('_statistics')
+  = render 'statistics/statistic', name: t('statistics.name'), value: @statistics.name, activated: "serv-enabled"
+  = render 'statistics/statistic', name: t('statistics.version'), value: @statistics.version, activated: "serv-enabled"
+  = render 'statistics/statistic', name: t('statistics.registrations'), value: registrations_status(@statistics), activated: "serv-enabled"
+  - if @statistics.expose_user_counts?
+    = render 'statistics/statistic', name: t('statistics.total_users'), value: @statistics.total_users, activated: "serv-enabled"
+    = render 'statistics/statistic', name: t('statistics.active_users_halfyear'), value: @statistics.halfyear_users, activated: "serv-enabled"
+    = render 'statistics/statistic', name: t('statistics.active_users_monthly'), value: @statistics.monthly_users, activated: "serv-enabled"
+  - if @statistics.expose_posts_counts?
+    = render 'statistics/statistic', name: t('statistics.local_posts'), value: @statistics.local_posts, activated: "serv-enabled"
+  - if @statistics.expose_comment_counts?
+    = render 'statistics/statistic', name: t('statistics.local_comments'), value: @statistics.local_comments, activated: "serv-enabled"
+.row-fluid
+  %h1
+    = t('statistics.services')
+  - Configuration::KNOWN_SERVICES.each do |service|
+    = render 'statistics/statistic', name: "#{service.capitalize}", value: service_status(service, @statistics.available_services), activated: service_class(service, @statistics.available_services)
```

Comment:
Did you get an error when requesting the mobile format, or why do you have an exact duplicate here?

If so, instead of duplicating the template, try rendering the same one for both formats:

```ruby
respond_to do |format|
  format.json { }
  format.any(:html, :mobile) { render 'statistics/statistics.html.haml' }
end